src/cfnlint/data/ExtendedSpecs/all/03_value_types.json

```diff
@@ -1,861 +1,887 @@
+          ]
+        }
+      },
+      "PlacementGroupStrategy": {
+        "AllowedValues": [
+          "cluster",
+          "partition",
+          "spread"
+        ]
+      },
+      "PlacementTenancy": {
+        "AllowedValues": [
+          "dedicated",
+          "default"
         ],
-        "Resources": [
-          "AWS::ElasticLoadBalancingV2::TargetGroup"
+        "Ref": {
+          "Parameters": [
+            "String"
+          ]
+        }
+      },
+      "PredicateType": {
+        "AllowedValues": [
+          "ByteMatch",
+          "GeoMatch",
+          "IPMatch",
+          "RegexMatch",
+          "SizeConstraint",
+          "SqlInjectionMatch",
+          "XssMatch"
         ]
-      }
-    },
-    "TargetGroupArns": {
-      "GetAtt": {},
-      "Ref": {
-        "Parameters": [
-          "Strings"
+      },
+      "PrivateIpAddress": {
+        "GetAtt": {
+          "AWS::EC2::NetworkInterface": "PrimaryPrivateIpAddress"
+        },
+        "Ref": {
+          "Parameters": [
+            "String"
+          ]
+        }
+      },
+      "RdsInstanceMonitoringInterval": {
+        "AllowedValues": [
+          "0",
+          "1",
+          "5",
+          "10",
+          "15",
+          "30",
+          "60"
+        ]
+      },
+      "RdsInstanceType": {
+        "Ref": {
+          "Parameters": [
+            "String"
+          ]
+        }
+      },
+      "RecordSetFailover": {
+        "AllowedValues": [
+          "PRIMARY",
+          "SECONDARY"
+        ]
+      },
+      "RecordSetGeoContinentCode": {
+        "AllowedValues": [
+          "AF",
+          "AN",
+          "AS",
+          "EU",
+          "NA",
+          "OC",
+          "SA"
+        ]
+      },
+      "RecordSetType": {
+        "AllowedValues": [
+          "A",
+          "AAAA",
+          "CAA",
+          "CNAME",
+          "MX",
+          "NAPTR",
+          "NS",
+          "PTR",
+          "SOA",
+          "SPF",
+          "SRV",
+          "TXT"
+        ]
+      },
+      "RedshiftClusterType": {
+        "AllowedValues": [
+          "multi-node",
+          "single-node"
+        ]
+      },
+      "RedshiftClusterVersion": {
+        "AllowedValues": [
+          "1.0"
+        ]
+      },
+      "RedshiftInstanceType": {
+        "Ref": {
+          "Parameters": [
+            "String"
+          ]
+        }
+      },
+      "Region": {
+        "AllowedValues": [
+          "af-south-1",
+          "ap-northeast-1",
+          "ap-northeast-2",
+          "ap-northeast-3",
+          "ap-south-1",
+          "ap-southeast-1",
+          "ap-southeast-2",
+          "ca-central-1",
+          "eu-central-1",
+          "eu-north-1",
+          "eu-west-1",
+          "eu-west-2",
+          "eu-west-3",
+          "sa-east-1",
+          "us-east-1",
+          "us-east-2",
+          "us-gov-east-1",
+          "us-gov-west-1",
+          "us-west-1",
+          "us-west-2"
+        ]
+      },
```

Comment:
missing:

```json
"ap-east-1",
"cn-north-1",
"cn-northwest-1",
"eu-south-1",
"me-south-1",
```

[removed these](https://github.com/aws-cloudformation/cfn-python-lint/pull/1486/commits/e5b62f2612ddbbb8371b79fe63e0373b256876c5#diff-8c17898a9115538e8716f153e1c7948f) hoping we could re-use [`helpers.py`](https://github.com/aws-cloudformation/cfn-python-lint/blob/master/src/cfnlint/helpers.py) instead